projects/igniteui-angular/src/lib/simple-combo/simple-combo.component.ts

```diff
@@ -212,12 +219,8 @@ export class IgxSimpleComboComponent extends IgxComboBaseDirective implements Co
 
     /** @hidden @internal */
     public writeValue(value: any): void {
-        const oldSelection = super.selection;
-        this.selectionService.select_items(this.id, this.isValid(value) ? [value] : [], true);
-        this.cdr.markForCheck();
-        this._displayValue = this.createDisplayText(super.selection, oldSelection);
-        this._value = this.valueKey ? super.selection.map(item => item[this.valueKey]) : super.selection;
-        this.filterValue = this._displayValue?.toString() || '';
+        this._value = value !== undefined ? [value] : [];
+        this.setSelection(new Set(this._value), undefined, false, false);
```

Comment:
Should we check if we also have the data loaded here, like we do with the other combo 🤔 In which case, it could be possible to have the `writeValue` method cease being abstract in `combo.common` as we could merge the logic of both methods into a common one.